tests/lib/devicetree/api/src/main.c

```diff
@@ -280,7 +280,6 @@ static void test_bus(void)
 	const char *i2c_dev = "TEST_I2C_DEV";
 	const char *spi_bus = "TEST_SPI_CTLR";
 	const char *spi_dev = "TEST_SPI_DEV";
-	const char *gpio = "TEST_GPIO_";
```

Comment:
We need to preserve test coverage for all macros regardless of whether they are deprecated or not, so this should be using the same method as f9c6ee77cc2df384bb8e4a8203791f0d12ffe041 to keep these tests here until the APIs are removed.